### DISCUSSION TITLE
Download Over VPN

### DISCUSSION THREAD

**Discussion by Balakumaransr** [grahampugh/erase-install#96] [Q&A]
Hi, 

Nice workflow .. I found your script will be single solution for upgrade, reinstall and eraseinstall and trying to leverage it .. We are ON-prem with Jamf and I got few questions/suggestions .

Preferring to install the package via recurring check-in 
1. When exactly the scripts start downloading the installer ?
2. If users are on VPN and disconnected , will the caching be interrupted ?
3. Since caching will be slow on VPN for sure please suggest how to make this work successfully ..

**Comment by grahampugh** [comment]
As I said in Slack, I do not recommend using a recurring check-in policy. The script is written with notifications built in, so users get a prompt when the download is happening. If that was to happen unexpectedly to a user, they would not be happy.

You might want to try using VPP to distribute the app. If you do this, you have to ensure that you restrict the installer app from being run, since it will otherwise open when it is downloaded.
